project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject kubernetes-api "0.3.3-SNAPSHOT"
```

Comment:
The CHANGELOG entry is missing as well...

Reply:
Since this is not a project of ours the CHANGELOG doesn't follow our pattern.